.travis.yml

```diff
@@ -14,8 +18,17 @@ env:
   - secure: dl8g/OwandAEZ5BASRf5vAsMldt8NpVSPQ9qICs6Ebz14WkJ2r7i6zR2WVj1gS+WdAGjFNydUKvMf6VRb0BAFto4WVmJi4lYzELfcHb87YUNFFNJc7rtOcbKgazz54vIkhe0VhELlGq9P0WAYf11xCThnBcNXthodvIdYNe3mSHSjvPL4ArdjE3DrBURqOY0MC2O8uOcX721fjlpDFZXWmvBtYub95czweZAet195PzzZA+jzoJe7Jn+NsYFRMGiIo58jvfyCxkwU3AR/TpcdnICj6JWoUifVP/lzEvxvzXIVIFt+M2zgybBclQyXO465cmP88omdM71v4f0hfW7Gjw85LYvCIuUIuo8A02jMXdy97eQZICOyoujFruDIcVMYBymJAU2NmpODGbDC3cYpxKjFgupzruQW5ELkJkD31YpRtbnm/Slxg+tbv8IZFNxFJDPrbvOKV4bJ1SJ9EcI0x+OF021scO6iZP+n1G3iScR7Jy8oKN+IfT6VaSAR/8dyeG0g6lNmU067MWBFPKYYqkixL8hQh5zZBtmqwcxJHDOtB81R7kT6jLvoxjquNypXQoupot/x28jIE0My7a+qTBlN5T4TxYhTBKqyMFV6Rk1ZbfiGO81Y0FvQr6WYJR/0vsxx8Whhteb1/YrYkdwkkMzO9rRET9H/2+isblXpm4=
   - secure: Dt5L2m6fMqwOlgMNqDyt20r+3DVP9hUZA9O9T5SoIqek1FSfnD5yh9l/XVfb0ylujYfIJ8dAFyCS20N8ZaWp84QbpcQUfI6U2ESpLG/KQwg2FiD5LoVdpmcSjj5D0ZIpYRDdNWmaW/NGHmWN5/eu/k33c73uNuOpwwYRzpwhyMUkXdLvzhGIDk/xZqUZOp+n4OLPJebnwxSzw/sF5DNtFigYqq+DTcOzg4SMsR35aYkSuuyw4axbFp5/zZIUdjsf4XbLT/pvZv0Oiwihk5aJ35qVnI5/R332EuU3fQrhNmPL6Qa2ByNXQRCsaKgj6eVcJretS/Ve/FaiPy3T3BBkv4/oleKSG1Jvcmfew+jvQT+LTA21T8OWak0A5uF2wHuZl/Ab/zdG9fc1+1Do/4h4fXVBQCQPuVcxDa3PZP+pgiRJmc33vAqU1A5jrElLG6tximFWRWaVPdlmapZgLBUFT6UwE9/IsRqFb1tRIXLVO6pn/8/WtzQfWGAdB2UUteBZ5WwEHSk8xiB2K7pQ0C6sGUXqHXt+tLbk1mVf/4NDxFILRGMmmFW1kb3o1oLeVBvzcD8Jt/SPyswqnGWv5PQ37qXw5p7WxU19duTJgq1X2iMjl7A7WVuaxYGCWciKFzKujdbpFPIYVS5GDHRzellsxSXPgJedGmc59yJ3opNwxEk=
   - secure: vH6+yZ54ig5Q9BHDIcHvikry+/HnyLHF8CwAUqhGD+f2o9M0UGP9bBKm0pD+u3xg9ugV9ZRNbDx08izU4ayspp61kKWptBy4TSZM2LPsKsduc6lwG9rIGbsTDJkUmHdiLAWSsw1hqEvt42zHjUSe2/s3v7Ai82pR/t9CppGSgtZb3tcusOOdd7+9rZBHdcrlZsfN7ZE75O+E4EI99537wFDFjY4PCD8QmuNGeYQB5SFZr00vElsVKjpTzf6qEo6ItYuCBoB0TV0IWEGUYPfZUsEQaXM6NpLTPj6NTnHiFWI6qmd0UrOV/WAseg+uSL24hmIlOe5Z/MIINev0q1dMpbAlVzSUT3oIuhlTeSs/isxDG8BD+1C4/g7ehlu31JG4mIXz1szEibQJDoGx0447+GhUP5TuemYXftxzTLQwTpRVueVAYwBgCEbSlHT4mtxs8ofZrVOm7nry2b0CtGoIynZ3WAVZ5/1nIMR6uBKfu3TNtzNK+WWBE3divcfyu9M82jV5XDFOgAKis48N8kMqbn6VgS11FXxJ0iSnxFJabLul64tu5/D1jbjEakTA7XquHazgk4LTCsuL8kGPxiZRFzovrucaH9hR5k5fSfoJ0sR2mNYRK0sGNNPtK7ZFwvEjU5CAJ0HIOFiBcUpfZcOv9WpdoreusrTz1mqAf66bFZc=
+
+before_install:
+- docker build -t PokemonGoers/PokeData .
```

Comment:
Not a valid tag. Name components may contain lowercase characters, digits and separators. A separator is defined as a period, one or two underscores, or one or more dashes. A name component may not start or end with a separator.
See [here](https://github.com/docker/docker/blob/master/docs/reference/commandline/tag.md)